profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info

```diff
@@ -0,0 +1,260 @@
+features[field_instance][] = taxonomy_term-newsroom_service-field_newsroom_service_image
+features[field_instance][] = taxonomy_term-newsroom_service-field_newsroom_topic
+features[field_instance][] = taxonomy_term-newsroom_service-field_service_active
+features[field_instance][] = taxonomy_term-newsroom_service-field_service_type
+features[field_instance][] = taxonomy_term-newsroom_service-name_field
+features[field_instance][] = taxonomy_term-newsroom_topic-field_newsroom_category_domain
+features[field_instance][] = taxonomy_term-newsroom_topic-field_newsroom_topic_id
+features[field_instance][] = taxonomy_term-newsroom_topic-name_field
+features[node][] = newsroom_item
+features[node][] = newsroom_selection
+features[taxonomy][] = newsroom_item_type
+features[taxonomy][] = newsroom_service
+features[taxonomy][] = newsroom_topic
+features[user_permission][] = administer newsroom advanced settings
+features[user_permission][] = administer newsroom settings
+features[user_permission][] = create newsroom_item content
+features[user_permission][] = delete any newsroom_item content
+features[user_permission][] = delete own newsroom_item content
+features[user_permission][] = edit any newsroom_item content
+features[user_permission][] = edit own newsroom_item content
+features[variable][] = additional_settings__active_tab_newsroom_item
+features[variable][] = comment_anonymous_newsroom_selection
+features[variable][] = comment_default_mode_newsroom_selection
+features[variable][] = comment_default_per_page_newsroom_selection
+features[variable][] = comment_form_location_newsroom_selection
+features[variable][] = comment_newsroom_selection
+features[variable][] = comment_preview_newsroom_selection
+features[variable][] = comment_subject_field_newsroom_selection
+features[variable][] = date_format_date_only
+features[variable][] = date_format_minimal
+features[variable][] = date_format_short_newsroom
+features[variable][] = field_bundle_settings_node__newsroom_item
+features[variable][] = field_bundle_settings_node__newsroom_selection
+features[variable][] = i18n_sync_node_type_newsroom_selection
+features[variable][] = language_content_type_newsroom_item
+features[variable][] = language_content_type_newsroom_selection
+features[variable][] = menu_options_newsroom_item
+features[variable][] = menu_options_newsroom_selection
+features[variable][] = menu_parent_newsroom_item
+features[variable][] = menu_parent_newsroom_selection
+features[variable][] = node_options_newsroom_item
+features[variable][] = node_options_newsroom_selection
+features[variable][] = node_preview_newsroom_item
+features[variable][] = node_preview_newsroom_selection
+features[variable][] = node_preview_page
+features[variable][] = node_submitted_newsroom_item
+features[variable][] = node_submitted_newsroom_selection
+features[variable][] = pathauto_node_newsroom_item_pattern
+features[views_view][] = newsletter_subscription
+features[views_view][] = newsroom
```

Comment:
"Newsroom" views - this is main page of the newsroom. It displays all newsroom items. The items can be filtered by type and topic.